clearml_agent/commands/worker.py

```diff
@@ -2919,7 +2922,11 @@ def install_requirements_for_package_api(
          """
         if package_api:
             package_api.cwd = cwd
-        api = self._install_poetry_requirements(repo_info)
+
+        lockfile_path = execution.working_dir if package_api.session.config["agent.package_manager.poetry_files_from_cwd"] \
+                            else ""
+        lockfile_path = Path(repo_info.root) / lockfile_path
+        api = self._install_poetry_requirements(repo_info, lockfile_path)
```

Comment:
Also, not sure why there are two terminologies - `lockfile_path` here and `install_path` as the function argument name - perhaps just choose one?

Reply:
Yes agreed, it was an artifact of something I tried before and didn't propagate all the naming. Fixed now 👍